src/test/java/codeu/model/store/persistence/PersistentStorageAgentTest.java

```diff
@@ -72,4 +77,13 @@ public void testWriteThroughMessage() {
     persistentStorageAgent.writeThrough(message);
     Mockito.verify(mockPersistentDataStore).writeThrough(message);
   }
+
+  @Test
+  public void testWriteThroughActivity() {
+    Activity activity =
+            new Activity(
+                    UUID.randomUUID(), UUID.randomUUID(), Action.REGISTER_USER, true, Instant.now(), null);
```

Comment:
What is true? Maybe we can do /** isPublic = */true

Reply:
Alright